templates/parachain/runtime/src/lib.rs

```diff
@@ -7,6 +7,8 @@
 include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 
 pub mod apis;
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarks;
```

Comment:
Why is this removed?

Reply:
Moved it up the top with the rest of the module declarations